### PR TITLE
Fix beginner's doc validation set issue

### PR DESCRIPTION
**Pull request by oke-aditya**
- This fixes the issue [#38735](https://github.com/tensorflow/tensorflow/issues/38735).

File changed:- beginner.ipynb.
- Replaced the validation set to test set. (This created the confusion).
- Also updated the history object. So that in next tutorials people get comfortable with plotting it.

### CODE REVIEW

**Comment by oke-aditya** [comment]
I did not know about CI for the notebook formatting. I have run it and that CI has passed now :+1:

**Comment by MarkDaoust** [comment]
Hi @oke-aditya, 

Thanks for taking the time to make a PR.

We're trying to keep this as short as possible, so I'd rather not mention features that aren't necessary (History).

(Also we try to avoid using "we" in these docs. )

For tensorflow/tensorflow#38735 I'd rather just say "validation set or test set": #1549.

Thanks again.
I'm closing this, please don't take it the wrong way.

**Comment by oke-aditya** [comment]
Thanks for that @MarkDaoust I would keep the above things in my mind next time I raise a PR / address an issue. It was my first experience in open-source so sorry for any mistakes.

**Comment by oke-aditya** [comment]
Can I delete this branch? as #1549 fixes it :+1:

**Comment by MarkDaoust** [comment]
> Can I delete this branch? 

Yes, when a pr is closed you can do whatever you like with the branch.
